### PR TITLE
Updated scoring.func.php file

### PR DESCRIPTION
**Pull request by madihamallick**
(Thanks for sending a pull request! Please make sure you click the link above to view the contribution guidelines, then fill out the blanks below.)

### Any relevant logs, error output, etc?
(If it’s long, please paste to https://ghostbin.com/ and insert the link here.)

### Any other comments?
....

### CODE REVIEW

File: functions/scoring.func.php [hunk 1]
```diff
@@ -1,7 +1,6 @@
 <?php
 /**
  * Undocumented function
- *
```
Comment (siv2r) [inline functions/scoring.func.php:4]: do not remove these lines. Spacing is good

Comment (siv2r) [reply functions/scoring.func.php:4]: It was generated automatically through the extension

File: functions/scoring.func.php [hunk 2]
```diff
@@ -71,15 +67,15 @@ function zeroHlaMismatch( $pair1, $pair2 ) {
...
- * @return integer
- */
 function highPRA() {
   return 0;
 }
-
+/**
+ * Undocumented function
+ * @param string $pair1
```
Comment (siv2r) [inline functions/scoring.func.php:75]: Spacing seems to be missing here...

**Review by siv2r — CHANGES_REQUESTED** [review]

**Comment by madihamallick** [comment]
okay I'll make the changes in a new PR :)

**Pull request closed without merging**